youtube_dl/extractor/cbsnews.py

```diff
@@ -60,7 +75,7 @@ def _real_extract(self, url):
         webpage = self._download_webpage(url, video_id)
 
         video_info = self._parse_json(self._html_search_regex(
-            r'(?:<ul class="media-list items" id="media-related-items"><li data-video-info|<div id="cbsNewsVideoPlayer" data-video-player-options)=\'({.+?})\'',
+            r'(?:<ul class="media-list items" id="media-related-items"[^>]+><li data-video-info|<div id="cbsNewsVideoPlayer" data-video-player-options)=\'({.+?})\'',
```

Comment:
Good catch.  Please see the updated regex - it should now include the exact match as well.  Thanks!